inference/worker/utils.py

```diff
@@ -64,6 +64,19 @@ def finish(self, reason: Literal["length", "eos_token", "stop_sequence"]) -> Ite
             yield from self.tokens
 
 
+def add_prefix_to_prompt(prompt: str):
```

Comment:
I think this function name is slightly misleading

Reply:
I agree. I changed it to `add_prefix_if_not_present` in the latest commit.